### PR TITLE
feat(footer): Add footer with RH logo and privacy policy

### LINKED ISSUE #13: Add Red Hat privacy policy and logo to footer
To support Adobe Analytics we need to display the RH privacy policy and logo. It can be placed in the footer similar to quarkus.io

### PR DESCRIPTION
**Pull request by jiekang**
Fixes #13

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -22,18 +22,22 @@ title: Cryostat
 author: The Cryostat Authors
 email: cryostat-development@googlegroups.com
 description: >- # this means to ignore newlines until "baseurl:"
-  JDK Flight Recorder for Kubernetes
+  JDK Flight Recorder Management for Kubernetes and OpenShift
 baseurl: "" # the subpath of your site, e.g. /blog
 url: "https://cryostat.io" # the base hostname & protocol for your site, e.g. http://example.com
 #twitter_username: 
 github_username: cryostatio
+github_fork_url: "https://github.com/cryostatio/cryostat"
```
Comment (andrewazores) [inline _config.yml:30]: Where is this used?

File: _includes/head.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  {%- seo -%}
+  <link rel="stylesheet" href="{{ "/assets/css/main.css" | relative_url }}">
+  {%- feed_meta -%}
```
Comment (andrewazores) [inline _includes/head.html:7]: What is this for - an RSS feed?

File: _layouts/default.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="{{ page.lang | default: site.lang | default: "en" }}">
+
+  {%- include head.html -%}
+
+  <body>
+
+    {%- include header.html -%}
```
Comment (andrewazores) [inline _layouts/default.html:8]: Are header.html and footer.html generated somehow or planned to be added? I don't see them.

File: _includes/head.html (hunk 2 above)
Comment (jiekang) [reply _includes/head.html:7]: Yes. The content of head.html is replacing the default head.html from minima, e.g. https://github.com/jekyll/minima/blob/master/_includes/head.html

I can investigate removal of feed_meta, etc. for our site

**Comment by andrewazores** [comment]
Is this the default font size/weight? The footer text is very fine.

image

It doesn't display the visual artifacts I see in the software screenshot, but on my display (24" 1920x1200) the text strokes are so fine that I'm seeing subpixel colouration on them. I'll upload an actual photo as well, if I can capture it.

File: _layouts/default.html (hunk 3 above)
Comment (jiekang) [reply _layouts/default.html:8]: The site uses minima layout so everything starts from their content, unless we explicitly override it. Those files would come from minima at this time.

https://github.com/jekyll/minima/

**Comment by jiekang** [comment]
> ~Is this the default font size/weight? The footer text is very fine.~
> 
> image
> 
> ~It doesn't display the visual artifacts I see in the software screenshot, but on my display (24" 1920x1200) the text strokes are so fine that I'm seeing subpixel colouration on them. I'll upload an actual photo as well, if I can capture it.~
> 
> Nevermind, it looks like that is just how it's styled by default - it's about the same on quarkus.io:
> 
> image
> 
> I do notice that the "Sponsored by Red Hat" is all the way to the right corner in our footer, whereas for Quarkus it has some margin on the right side still. Should ours be the same? Both of those screenshots extend all the way to the lower-right corner of my display area.

I think the quarkus site has a default margin on left and right sides that we might be missing. I'll look into it. I tried not to import all of their core scss things as they also affected the sizes for everything else on our page.

File: _config.yml (hunk 1 above)
Comment (jiekang) [reply _config.yml]: Removed. It's unused for now. If it ever is needed, we can add it then.

**Comment by jiekang** [comment]
@andrewazores 
I've added rem based padding to the footer now.

**Review by andrewazores — APPROVED** [review]

**Pull request merged**